plugins/lookup/random_string.py

```diff
@@ -25,26 +25,30 @@
     type: int
   upper:
     description:
-      - Include uppercase letters in the string.
+      - Possibly include uppercase letters in the string.
+      - To ensure atleast one uppercase letter, set C(min_upper) to V(1).
```

Comment:
```suggestion
      - To ensure atleast one uppercase letter, set O(min_upper) to V(1).
```
Same for the others.